orion-ui/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@prefecthq/orion-design": "1.1.6",
-    "@prefecthq/prefect-design": "1.1.29",
+    "@prefecthq/prefect-design": "^1.1.32",
```

Comment:
To check my own memory, some time ago there was a reason we weren't using `^` for prefecthq packages. Is it still a thing, or I'm misremembering? @pleek91 @znicholasbrown

Reply:
Yes, definitely still a thing - this is to prevent unintentional version bumps that might contain breaking changes

Reply:
If its a `dependency` it should be exact. If its a `peer` it should have the `^` is the general rule of thumb for packages.